.travis.yml

```diff
@@ -71,18 +69,28 @@ install:
   - echo "CCUSTOMOPTIONS='-DCMAKE_C_COMPILER_LAUNCHER=ccache -DCMAKE_CXX_COMPILER_LAUNCHER=ccache -DCMAKE_C_FLAGS=\"-Werror\" -DCMAKE_CXX_FLAGS=\"-Werror\"'" >> conf/config.sh
   - echo "DB_CHARACTERS_CONF=\"MYSQL_USER='root'; MYSQL_PASS=''; MYSQL_HOST='127.0.0.1';\"" >> conf/config.sh
   - echo "DB_AUTH_CONF=\"MYSQL_USER='root'; MYSQL_PASS=''; MYSQL_HOST='127.0.0.1';\"" >> conf/config.sh
-  - echo "DB_WORLD_CONF=\"MYSQL_USER='root'; MYSQL_PASS=''; MYSQL_HOST='127.0.0.1';\"" >> conf/config.sh  
-  # create and import mysql
-  - mysql -e "SET GLOBAL sql_mode = '';" # this is necessary because of mysql 5.7
-  - bash ./acore.sh "db-assembler" "import-all"
+  - echo "DB_WORLD_CONF=\"MYSQL_USER='root'; MYSQL_PASS=''; MYSQL_HOST='127.0.0.1';\"" >> conf/config.sh
+  - RND_NAME=$(cat /dev/urandom | tr -dc 'a-z' | fold -w 5 | head -n 1)
+  - echo "DB_AUTH_NAME=auth_$RND_NAME" >> conf/config.sh
+  - echo "DB_CHARACTERS_NAME=characters_$RND_NAME" >> conf/config.sh
+  - echo "DB_WORLD_NAME=world_$RND_NAME" >> conf/config.sh
 
-script:
+before_script:
   # compile
   - export CCACHE_CPP2=true
   - ccache -s
   - timeout 2580 bash ./acore.sh "compiler" "all"
   - ccache -s
-  - git clone --depth=1 --branch=master --single-branch https://github.com/ac-data/ac-data.git /home/travis/build/azerothcore/azerothcore-wotlk/env/dist/data
-  - cp ./data/travis/worldserver.conf ./env/dist/etc/worldserver.conf
-  - ./env/dist/bin/worldserver --dry-run
-  - ./apps/ci-error-check.sh
+
+jobs:
+  include:
+    - stage: db_test
+      env:
+        - TRAVIS_BUILD_ID="1"
```

Comment:
Why do we need this again?

Reply:
I wanted the job to be executed just for one build, but this doesn't work (it overrides the matrix and executes only this single job). I'm just playing around with Travis, still new to this.